tests/test_inference_costs.py

```diff
@@ -0,0 +1,1260 @@
+#!/usr/bin/env python3
+"""Test script for incentives integration and costs checks. NOT comprehensive test suite yet."""
+
+import sys
+from pathlib import Path
+
+import pandas as pd
+
+sys.path.insert(0, str(Path(__file__).parent.parent))
```

Comment:
not sure why this is necessary, probably for the import below to work.  

if you were having trouble running it as a script, you should be able to use `uv run path/to/file.py`